fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/HeadersJMSIT.java

```diff
@@ -131,7 +134,8 @@ public void testIngestion() throws RepositoryException {
         try {
             containerService.findOrCreate(session, testIngested);
             session.save();
-            awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE, null);
+            final Message ingestMsg = awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE);
+            assertTrue(getResourceTypes(ingestMsg).contains("fedora:Resource"));
```

Comment:
My two priorities here are:

  1. make any resource types coming out of the JMS broker to be unambiguous. To me, that means they are fully qualified or a namespace mapping is provided _with every message_. I am strongly in favor of the former approach.
  2. keep the JMS module independent of JCR and the modeshape implementation; that way it could be re-used by an alternate implementation.

Both of these can very easily be addressed by simply making those qnames (`fedora:Resource`) into fully qualified URIs (`http://fedora.info/definitions/v4/repository#Resource`) inside the modeshape implementation code.